docs/learn/avalanche/avalanche-consensus.md

```diff
@@ -1,45 +1,81 @@
 ---
-description: Consensus is the task of getting a group of computers to come to an agreement on a decision. Avalanche is a new consensus protocol that is scalable, robust, and decentralized.
+description: Avalanche is a new consensus protocol that is scalable, robust, and decentralized.
 ---
 
 # Avalanche Consensus
 
-
-Consensus is the task of getting a group of nodes to come to an agreement on a decision.
-Nodes can reach a consensus by following a set of steps called a consensus protocol. Avalanche
-is a new consensus protocol that is scalable, robust, and decentralized. It has low latency and high
-throughput. It is energy efficient and does not require special computer hardware. It performs well
-in adversarial conditions and is resilient to "51% attacks." This document explains the Avalanche
-consensus protocol. The white-paper is [here](https://www.avalabs.org/whitepapers).
+Consensus is the task of getting a group of computers (a.k.a. nodes) to come to an agreement on a decision.
+In blockchain, this means that all the participants in a network have to agree on the changes made to
+the shared ledger. This agreement is reached through a specific process, a consensus protocol,
+that ensures that everyone sees the same information and that the information is accurate and trustworthy.
+
+**Avalanche Consensus** is a new consensus protocol that is scalable, robust, and decentralized. It
+combines features of both classical and Nakamoto consensus mechanisms to achieve high throughput, 
+fast finality, and
+energy efficiency. For the whitepaper, see [here](https://www.avalabs.org/whitepapers). 
+
+Key Features Include: 
+
+- Speed: Avalanche consensus provides sub-second, immutable finality, ensuring that transactions are
+quickly confirmed and irreversible.
+- Scalability: Capable of a peak throughput of up to 20,000 transactions per second with a latency
```

Comment:
> Capable of a peak throughput of up to 20,000 transactions per second

On [the 5-23-23 Dev Community call, @ 21:29](https://www.youtube.com/live/HD3vdZzraE8?feature=share&t=1289), POG announced 50K TPS